src/define_plugin_sets.h

```diff
@@ -23,6 +23,7 @@ To create/register a plugin, you have to :
 */
 
 //#define FEATURE_SD
+#define FEATURE_I2CMULTIPLEXER
```

Comment:
```c++
#ifndef FEATURE_I2CMULTIPLEXER
    #define FEATURE_I2CMULTIPLEXER
#endif
```

Reply:
You undefine it also in the BUILD_ONLY_SWITCH, but the other entries there only 'add' stuff.
So I guess it should be defined in the 'normal' build and undefined when space is needed.
This also allows it to be built using Custom builds.

Reply:
I'll fix that part, I now see it should be done somewhat different. Thnx.

Reply:
I'm thinking of *only* setting this define from the environment configuration files, as I kind of deemed it as a feature, that needs to be enabled explicitly. And then I'd set it only for the 4 MB builds.

Reply:
Just keep in mind these 2 simple "rules":
- Everything must be configurable from a "custom" build.
- Builds aimed for limited resources generally use blocks to specifically "disable" parts.

If you keep those in mind, I guess it can be rather simple to decide where to put this check.
Just make sure to wrap it in #ifndef like I showed, since it could already be defined explicitly.

Reply:
I now only define it in the `PLUGIN_BUILD_NORMAL` configuration, and undef it in the `BUILD_MINIMAL_OTA` or `MEMORY_ANALYSIS` special configuration, so it is effectively included in most normal builds. Probably some that now go over their size need this to be undef'ed, we'll see what builds travis chokes on.